pom.xml

```diff
@@ -86,7 +86,9 @@
     <version.org.jboss.spec.javax.transaction>1.0.0.Alpha3</version.org.jboss.spec.javax.transaction>
     <version.org.jboss.spec.javax.resource.jboss-connector-api_1.7_spec>1.0.0.Final</version.org.jboss.spec.javax.resource.jboss-connector-api_1.7_spec>
     <version.junit>4.11</version.junit>
-    <version.narayana>5.3.4.Final</version.narayana>
+    <version.narayana>5.5.2.Final</version.narayana>
+    <maven.compiler.source>1.8</maven.compiler.source>
```

Comment:
In an earlier version I was using the Java 8 "default methods" feature. I removed that usage so I can drop the 1.8 requirement.